### DISCUSSION TITLE
お知らせの望ましい用途について

### DISCUSSION THREAD

**Discussion by tai-cha** [misskey-dev/misskey#14033] [Q&A]
Misskey 2024.5.0時点でお知らせの設定画面には、「特に新規ユーザーのUXを損ねる可能性が高いため、常時掲示するための情報ではなく、即時性が求められる情報の掲示のためにお知らせを使用することを推奨します。」とあります。

新規・既存を問わないユーザー向けの常時掲示したい情報はどこに載せるのが適切なのでしょうか？

![image](https://github.com/misskey-dev/misskey/assets/40626578/336c0600-430c-4809-a62c-a524da64bd5f)

参考情報: この文言が追加されたコミット https://github.com/misskey-dev/misskey/pull/12177/commits/ee191169f5ed90b9a343d21eaf2166ffd71efebc

**Comment by syuilo** [comment]
特にないかも

**Comment by syuilo** [comment]
具体的にどのような情報かにもよりそう

**Comment by syuilo** [comment]
例えば利用にあたっての注意事項とかお願いであれば利用規約に書くべきと思う

**Reply by tai-cha** [reply]
例えば利用規約以外のガイドラインとか？
ioで言うところのNSFW基準みたいなドキュメントとか、あるいはサーバー公式Discordとか

**Comment by tai-cha** [comment]
例えばサーバールール、利用規約と別に投稿ガイドラインとかNSFWガイドラインを置く場合とかですね(この場合だと利用規約からリンクしろって感じかしら)

**Reply by Sayamame-beans** [reply]
にりらみすきー部だと、ロールの情報公開やMisskey更新に伴って発生した(ユーザー側で対応が必要な)不具合情報の掲示、Discordサーバー等へのリンク集などを載せていますね…
